src/main/java/com/easypost/net/EasyPostResource.java

```diff
@@ -717,15 +669,15 @@ public String getLabelUrl() {
         return "";
     }
 
-    // Tracker
-
     /**
      * @return the ID of this shipment
      */
     public String getShipmentId() {
         return "";
     }
 
+    // Batch
```

Comment:
This comment was misplaced during the shuffle

Reply:
Resolved in 28156240aa91125dfeeb163cd5eb6d71869b762a